lib/widgets/map_widget.dart

```diff
@@ -16,11 +16,25 @@ import 'package:concordia_navigation/models/buildingModels/building_information.
 
 //This is the map widget that will be loaded in the home screen.
 class MapWidget extends StatefulWidget {
+  final double _expandedBottomSheetBottomPosition = 0;
+  final double _collapsedBottomSheetBottomPosition = -260;
   @override
   _MapWidgetState createState() => _MapWidgetState();
 }
 
 class _MapWidgetState extends State<MapWidget> {
+  double _bottomSheetBottomPosition = -260;
+  bool isCollapsed = false;
+
+  _onTap() {
+    setState(() {
```

Comment:
Maybe we can extract this to the map data provider. 
That way we can pull up the directions list on click from a different UI component.
thoughts?

Reply:
Fixed